### PR TITLE
swp2clk - Implement compatibility with Bangle.load/Bangle.showClock

### PR DESCRIPTION
**Pull request by halemmerich**
This implements an additional path for firmware newer than 2v15.135 while for now keeping compatibility with older firmware.

### CODE REVIEW

**Comment by thyttan** [comment]
Something weird happens for me. I have Slope Clock, Quick Launch and Torch (last two with my newest changes in PRs) set up so:

Starting after an initialization of .bootcde:
1. swipe l-t-r starts torch with fast switching
2. swipe l-t-r shows the clock with fast switching
3. **swipe l-t-r shows the "Loading..." message after which the clock is shown again (<--- the strange part!)**
4. go to 1, loop ad infinitum.

**Comment by halemmerich** [comment]
Hm, something strange is happening while registering a swipe handler inside a swipe handler. It seems both handlers get executed, the first clears Bangle.uiRemove, and the second then has to normal load the app because no uiRemove method is set...

**Comment by halemmerich** [comment]
I think these commits fix the problem by adding the new swipe handler in the next idle. That should prevent the newly added handler running directly and creating the problem.

**Comment by thyttan** [comment]
Tested and works for me! Thanks 👌

**Comment by halemmerich** [comment]
This will probably fail on older firmware with polyfills, need to check that later this day.

File: apps/swp2clk/boot.js [hunk 1]
```diff
@@ -19,17 +19,19 @@
   var DEBUG = false;
   var FILE = "swp2clk.data.json";
 
+  var swipeHandler = (dir) => {
+    log("swipe");
+    log(dir);
+    if (dir === 1) {
+      if (!Bangle.showClock) load();
```
Comment (gfwilliams) [inline apps/swp2clk/boot.js:26]: Do we need this now? showClock should always be defined after the latest bootloader tweaks

Comment (halemmerich) [reply apps/swp2clk/boot.js:26]: If we assume that all apps are up to date, this is not needed.

**Comment by halemmerich** [comment]
This seems to work for me on 2v15 as well as 2v15.138 in combination with quicklaunch.

**Comment by gfwilliams** [comment]
Wow, this looks great - thanks! This is miles cleaner!

**Pull request merged**